web/wp-content/themes/dctx/single.php

```diff
@@ -5,9 +5,21 @@
 
 		<div class="main-content">
 			<?php while (have_posts()) : the_post(); ?>
-				<h1><?php the_title(); ?></h1>
+				<h1 class="post-title"><?php the_title(); ?></h1>
 				<?php dctx_post_meta(); ?>
 
+				<?php if (has_post_thumbnail()): ?>
```

Comment:
:no_entry_sign: **Error**: No space after opening parenthesis is prohibited (*WordPress.WhiteSpace.ControlStructureSpacing.NoSpaceAfterOpenParenthesis*).